### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by mauricerkelly** [powderworks/powderworks.github.io#3]
It'd be worth getting an accessibility review done to ensure that everyone, regardless of impairments, has an opportunity to use this page.

**Comment by CJKerr** [comment]
https://www.w3.org/standards/webdesign/accessibility
https://www.w3.org/TR/2008/REC-WCAG20-20081211/

https://achecker.ca/ 
This tool says that the link colours have insufficient contrast with the background (I have no opinion, usability wise, but I'm inclined to agree that they look bad aesthetically)

It also suggests that we should remove the deprecated bold and italic elements and use appropriately-styled versions of their semantic replacements em (I think this is italic in the default stylesheets) and strong (likewise for bold) instead.

Those are trivial changes, and I'll work up a branch momentarily.

**Comment by CJKerr** [comment]
For now I've retained the general colour of the links, just adjusted them to pastel variations with better contrast on the dark background. Live version here: https://cjkerr.github.io/powderworks.github.io/

Conversely, I've changed the styling of the horizontal break to be less conspicuous. I've maybe gone too far on that...

Various invisible accessibility changes have been made, mostly more semantic use of html elements.

I've also adjusted the spacing of elements to give a consistent vertical rhythm. Mostly a personal preference, because I don't think most people notice.

**Comment by mauricerkelly** [comment]
I think it's looking great. Definitely worth getting a PR going at this stage.

**Comment by CJKerr** [comment]
Sure - I have a small CSS fix to commit to fix the horizontal breaks messing up the vertical rhythm, then I'll branch and do a PR

**Comment by robmanthey** [comment]
You could slip in a non-breaking space in the Midnight Oil link while tidying. That's wrapping on a smartphone. Yuck.

**Comment by mauricerkelly** [comment]
That reminds me, there's a whole "responsiveness" issue to be addressed, but I figured that was for a different issue/PR :-)

**Comment by CJKerr** [comment]
Yeah, @robmanthey, good idea. I'll do it, but it should be a non-issue with a responsive layout. Was thinking it should flow Header -> Right Pane -> Left Pane -> Footer on mobile. That's simple enough, it will take longer to remove the existing table than it will to fix it up with box model elements. Another branch after I've done a PR for this one.

**Comment by robmanthey** [comment]
Loving watching your work by the way ...

**Comment by CJKerr** [comment]
Pull requests opened for both accessibility and layout.

Either merge accessibility first, or decline that pull and just pull the layout PR.

View my master branch here for these changes and whatever else I've finished in the meantime: https://cjkerr.github.io/powderworks.github.io/

**Comment by CJKerr** [comment]
OK, PR #5 was a good start at cleaning this up, but we should probably find someone more expert to take a look.

**Comment by mauricerkelly** [comment]
I think you've done enough to close off this issue though!

**Comment by robmanthey** [comment]
Closing issue. Add a new issue if a fresh look is inspired.
